lib/public/views/lhcPeriods/Overview/LhcPeriodsOverviewModel.js

```diff
@@ -109,7 +105,7 @@ export class LhcPeriodsOverviewModel extends Observable {
      * @returns {RemoteData} the remote data of LHC periods
      */
     get lhcPeriods() {
-        return this._lhcPeriods;
+        return this._dataSource.observableData.current;
```

Comment:
Seeing this makes me think even more that data source should not hold the observable. Here you are using datasource in a place I don't think it should, here you should only manipulate the actual data and not couple this method to the datasource.